canvas_modules/harness/src/intl/locales/en.json

```diff
@@ -3,5 +3,7 @@
   "canvas.emptyText": "Welcome to the Common Canvas test harness.{br}Your flow is empty!",
   "canvas.emptyLink": "Click here to take a tour",
   "expression.value.column": "Custom Value",
-  "expression.value.column.description": "Testing custom desc from app"
+  "expression.value.column.description": "Testing custom desc from app",
+  "password.showPasswordLabel.tooltip": "Show",
+  "password.hidePasswordLabel.tooltip": "Hide"
```

Comment:
Can you change one of these labels to show the test harness overriding the default label?

Reply:
Hi @caritaou , Im still trying to make `flyoutEditorFormWithInt` work since its calling 'mountWithIntlMessages()' to allow for custom resources to be overridden from the test harness, I'm assuming this is the only way to update labels from within harness ?

Reply:
Hey @nabilIbm, the harness is acting as an application overriding the resources from common-properties, so yea it is.
Did you take a look at the example I sent you? https://github.com/elyra-ai/canvas/blob/main/canvas_modules/common-canvas/__tests__/common-properties/controls/expression-test.js#L444

Reply:
Yes, i saw that commit...Im working on adjusting (something wrong still cant re-render the component with the new custom `label password.showPasswordLabel.tooltip: Show custom` using `flyoutEditorFormWithInt` )  this function parameter and making it work for my passwordControl just like the expression one